x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/policy_template_form.tsx

```diff
@@ -73,6 +79,114 @@ interface IntegrationInfoFieldsProps {
   onChange(field: string, value: string): void;
 }
 
+type AwsAccountType = 'single_account' | 'organization_account';
+
+const getAwsAccountTypeOptions = (isAwsOrgDisabled: boolean) => {
+  const awsAccountTypeOptions: CspRadioGroupProps['options'] = [
+    {
+      id: 'single_account',
+      label: i18n.translate('xpack.csp.fleetIntegration.awsAccountType.singleAccountLabel', {
+        defaultMessage: 'Single Account',
```

Comment:
nit: (or rather a question) if I remember correctly sentence case is used in Kibana texts, do I remember correctly? should it be `Single account` and `AWS organization` then?

Reply:
I'm not sure this is considered a sentence, but a title. AWS also refer to it the same way
![image](https://github.com/elastic/kibana/assets/51442161/d5a7b7a4-e68d-4295-b5f7-ba63123d01e7)

But ultimately I followed the designs. We can ping the design/product team if you think its needed, lmk :)

![image](https://github.com/elastic/kibana/assets/51442161/8ee58205-0b45-41ad-9d52-db0c61d37928)

Reply:
Thanks for the context, if it's like that in designs, we don't need to spend time investigating it further